MyProject/src/main/resources/dev.json

```diff
@@ -0,0 +1,92 @@
+﻿[{"nics":[
```

Comment:
We will not need this for now. Later we will think about the model and decide about more convinient way how to store test data structures.